data/obs-packaging/cc-oci-runtime/README.md

```diff
@@ -8,10 +8,14 @@ With these files we generated Fedora and Ubuntu packages for this component.
 
 ``./update_runtime.sh [VERSION]``
 
+The ``VERSION`` parameter is optional, it can be a tag, a branch or a git hash.
```

Comment:
The ``VERSION`` parameter is optional. The parameter can be a tag, a branch, or a GIT hash.